api/ocp/scc.yaml

```diff
@@ -0,0 +1,37 @@
+kind: SecurityContextConstraints
+metadata:
+  name: {{ .Name }}
+allowHostDirVolumePlugin: true
+allowHostIPC: true
+allowHostNetwork: false
+allowHostPID: true
+allowHostPorts: true
+allowPrivilegedContainer: true
+allowedCapabilities:
+  - SYS_ADMIN
+apiVersion: security.openshift.io/v1
+defaultAddCapabilities: []
+fsGroup:
+  type: RunAsAny
+priority:
+readOnlyRootFilesystem: false
+requiredDropCapabilities:
+  - ALL
+runAsUser:
+  type: RunAsAny
+seLinuxContext:
+  type: RunAsAny
+supplementalGroups:
+  type: RunAsAny
+users:
+  - system:serviceaccount:{{.Namespace}}:csi-rbd-ctrlplugin-sa
+  - system:serviceaccount:{{.Namespace}}:csi-cephfs-ctrlplugin-sa
+  - system:serviceaccount:{{.Namespace}}:csi-nfs-ctrlplugin-sa
+  - system:serviceaccount:{{.Namespace}}:csi-rbd-nodeplugin-sa
+  - system:serviceaccount:{{.Namespace}}:csi-cephfs-nodeplugin-sa
```

Comment:
Having a template here doesnt help upstream Rook users. we need complete yaml where Rook or user can use it directly

Reply:
+1